roles/contrail_node/tasks/main.yaml

```diff
@@ -0,0 +1,47 @@
+---
+- when:
+  - docker_registry is not defined
+
+  block:
+  - name: Untar Contrail tar-file
+    unarchive:
+      src: "{{ contrail_docker_images_path }}/contrail-kubernetes-docker-images_{{ contrail_version }}.tgz"
```

Comment:
This is a tarball containing two tarballs?

Does this match some vendor documentation?  Where is this tarball sourced?  I would like to see a link in the README.md that links to appropriate upstream documentation.